### PR TITLE
Volumn slider

### PR DESCRIPTION
**Pull request by Yahyakiani**
Added Volume Slider for the Mute functionality as well.

### CODE REVIEW

File: src/components/apps/dungeon/home.tsx [hunk 1]
```diff
@@ -196,8 +196,8 @@ export function DungeonApp()
 
 
     const handleBetChange = (selected : BetSize) => {
-        
-        if (!isMuted) wagerSelectAudio.play()
+        wagerSelectAudio.volume=volume/100
+        if (muteState!==1) wagerSelectAudio.play()
```
Comment (daoplays) [inline src/components/apps/dungeon/home.tsx:200]: in general please put spaces between logical/mathematical operators:
muteState !== 1

volume = volume / 100

etc

I wont leave a comment by each one but please go through

File: src/components/apps/dungeon/home.tsx [hunk 2]
```diff
@@ -986,6 +986,7 @@ export function DungeonApp()
                     setEnemyState(DungeonStatus.alive);
                     
                     //player death audio
+                    PlayerDeathAudio.volume=volume/100
                     playAudio(PlayerDeathAudio)
```
Comment (daoplays) [inline src/components/apps/dungeon/home.tsx:990]: why not change playAudio to include a volume, that way you dont need this line above each audio element, and it won't be necessary to remember to do this for future sound effects

File: src/components/apps/dungeon/home.tsx [hunk 3]
```diff
@@ -975,7 +975,7 @@ export function DungeonApp()
                     setEnemyState(DungeonStatus.dead);
         
                     //Victory sound plays
-
+                    VictoryAudio.volume=volume/100
```
Comment (daoplays) [inline src/components/apps/dungeon/home.tsx:978]: is there anywhere you don't do volume / 100?  can you not just do this before it is passed on

File: src/components/apps/dungeon/home.tsx [hunk 4]
```diff
@@ -1062,7 +1063,8 @@ export function DungeonApp()
     }, [set_JWT_token]);
 
     const handleEnterBtn = ()=>{
-        if (!isMuted) dungeonTileAudio.play()
+        dungeonTileAudio.volume=volume/100
+        if (muteState!==1) dungeonTileAudio.play()
```
Comment (daoplays) [inline src/components/apps/dungeon/home.tsx:1067]: can you make use of the playAudio function for these (and the others)

File: src/components/apps/dungeon/musicPlayer.tsx [hunk 5]
```diff
@@ -14,23 +15,29 @@ interface MusicTrack {
...
-  isMuted: boolean;
+  // muteState: number;
 }
 
-const MusicPlayer = ({ tracks,isMuted }: MusicPlayerProps) => {
+
+
+const MusicPlayer = ({ tracks }: MusicPlayerProps) => {
   const [audioSrc, setAudioSrc] = useState<string>(tracks[0].src);
+  const {  volume,muteState } = useContext(MuteContext);
```
Comment (daoplays) [inline src/components/apps/dungeon/musicPlayer.tsx:25]: volume, muteState

File: src/components/apps/dungeon/home.tsx (hunk 1 above)
Comment (Yahyakiani) [reply src/components/apps/dungeon/home.tsx:200]: got it

**Pull request merged**